pkg/config/merge.go

```diff
@@ -14,3 +19,10 @@ func mergeStringMap(intoStringMap, fromStringMap map[string]string) {
 		}
 	}
 }
+
+func mergePortBindings(intoPorts []PortBinding, fromPorts []PortBinding) {
```

Comment:
Where you planning on removing these `fmt.Println()`

Reply:
thanks